test/typescript/apis/project/cds-ql.ts

```diff
@@ -104,6 +99,30 @@ DELETE([Foo, Foos])
 
 let selectOne: Foo
 let selectMany: Foos
+
+// SINGULAR TESTS
+selectOne = await SELECT.from(Foos.drafts, 42) // .drafts of plural still singular
```

Comment:
this is actually an [open question](https://github.com/cap-js/cds-typer/pull/192) as of today. 🫠 But let's roll with it for now.